packages/rancher-istio/charts/configs/istio-base.yaml

```diff
@@ -34,187 +40,38 @@ spec:
     telemetry:
       enabled: {{ .Values.telemetry.enabled }}
   hub: docker.io/istio
-  meshConfig:
-    {{- if .Values.global.accessLogFile }}
-    accessLogFile: "{{ .Values.global.accessLogFile }}"
-    {{- end }}
-    {{- if .Values.global.disablePolicyChecks }}
-    disablePolicyChecks: {{ .Values.global.disablePolicyChecks }}
-    {{- end }}
-    defaultConfig:
-      proxyMetadata: {}
-    {{- if .Values.global.enableAutoMtls }}
-    enableAutoMtls: {{ .Values.global.enableAutoMtls }}
-    {{- end }}
   profile: default
-  tag: 1.6.5
+  tag: {{ .Values.global.tag }}
+  revision: {{ .Values.revision }}
   values:
+    gateways:
+      istio-egressgateway:
+        type: NodePort
```

Comment:
we should make this editable so people can use an LB if they want, but we default to NodePort